src/UI/Pages/Agents/AgentsTableRow.tsx

```diff
@@ -26,26 +25,24 @@ export const AgentsTableRow: React.FC<Props> = ({ row }) => {
                 id: row.process_id,
               })}
             >
-              <Button variant="tertiary" isSmall icon={<InfoCircleIcon />}>
-                {row.process_name}
-              </Button>
+              <Button variant="link">{row.process_name}</Button>
             </Link>
           )}
         </Td>
-        <Td dataLabel={words("agents.columns.status")}>
+        <Td width={10} dataLabel={words("agents.columns.status")}>
           <StatusLabel status={row.status} />
         </Td>
-        <Td dataLabel={words("agents.columns.failover")}>
+        <Td width={10} dataLabel={words("agents.columns.failover")}>
           {row.last_failover && (
             <DateWithTooltip timestamp={row.last_failover} />
           )}
         </Td>
-        <Td dataLabel={words("agents.columns.unpause")}>
+        <Td width={10} dataLabel={words("agents.columns.unpause")}>
           {row.unpause_on_resume !== null && row.unpause_on_resume !== undefined
             ? JSON.stringify(row.unpause_on_resume)
             : null}
         </Td>
-        <Td dataLabel={words("agents.columns.actions")}>
+        <Td isActionCell modifier="fitContent">
           <Actions name={row.name} paused={row.paused} />
```

Comment:
The Actions component should probably be split up. I know this is not really described in this ticket.
But the patternfly tables use a different `Td` for the action link and the kebap dropdown.
I would split up the actions component and put them in separate `Td`s. The link should be `fitContent`.
The kebap should be `isActionCell`.... if we want to do it like patternfly of course.